src/clipboard.ts

```diff
@@ -105,7 +105,7 @@ export async function copyTextToSystemClipboard(text: string | null) {
       //  not focused
       await navigator.clipboard.writeText(text || "");
       copied = true;
-    } catch (error) {}
+    } catch {}
```

Comment:
maybe logging the error instead of swallowing it?

Reply:
I can do that. The reason I didn’t was because it was being swallowed before.

Reply:
I would say so.. It's better for the long run :)